custom_components/atrea/climate.py

```diff
@@ -212,6 +214,8 @@ def extra_state_attributes(self):
         attributes["active_inputs"] = self._active_inputs
         attributes["forced_mode"] = self._forced_mode.name
         attributes["current_power"] = self._current_power
+        attributes["in1"] = self._in1
+        attributes["sa1"] = self._sa1
```

Comment:
Could we modify this please so that it is not exposed at all if it is None?

Reply:
Thanks, make sense. Fixed.